scss/os/_overrides_tuieditor.scss

```diff
@@ -102,18 +102,121 @@
       @include img-fluid;
     }
 
+    p {
+      color: color-yiq($u-theme-variables--bg-offset);
+      font-family: $font-family-sans-serif;
+      font-size: $font-size-base;
+      margin-bottom: 1rem;
+      margin-top: 0;
+    }
+
+    hr {
+      border: 0;
+      border-top: 1px solid rgba($white, .35);
+      margin-bottom: .2rem;
+      margin-top: .2rem;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      color: color-yiq($u-theme-variables--bg-offset);
+      margin-bottom: .5rem;
+      margin-top: 0;
+      padding-bottom: 0;
+    }
+
+    h1 {
+      border-bottom: 0;
+      font-size: 2.1875rem;
+      font-weight: normal;
+    }
+
+    h2 {
+      border-bottom: 0;
+      font-size: 1.75rem;
+      font-weight: normal;
+    }
+
+    h3 {
+      font-size: 1.53125rem;
+      font-weight: normal;
+    }
+
+    h4 {
+      font-size: 1.3125rem;
+      font-weight: normal;
+    }
+
+    h5 {
+      font-size: 1.09375rem;
+      font-weight: normal;
+    }
+
+    h6 {
+      font-size: .875rem;
+      font-weight: normal;
+    }
+
+    @media (max-width: 1200px) {
+      h1 {
+        font-size: calc(1.34375rem + 1.125vw);
```

Comment:
These are some very specific values. 1.34375 ?

Reply:
I couldn't find a variable or calculation for any of the header sizes, and actually ended up grepping them out of the `.build/themes` files to find what was used. They match what was compiled before the Toast UI upgrade. The problem I ran into is that the updated Toast UI has a newer stylesheet that mucks with a lot of our theming.